ItemLevelDownloader/package.json

```diff
@@ -9,8 +9,7 @@
   },
   "dependencies": {
     "handlebars": "4.0.10",
-    "minimatch": "3.0.2",
-    "azure-storage": "2.2.1"
+    "minimatch": "3.0.2"
```

Comment:
Please move shrinkwrap file as well. I had created this file for azure-storage only. Hence you can move it completely. @omeshp have you changed that file? If so, then it please retain your changes. I guess you will need another list once Sriram gets OSS approval.

Reply:
As shrinkWrap is not present in this Repo. i shall use shrink wrap in vsts-tasks.